nodejs/week1/meal_sharing_website/src/backend/routes/largeMeals.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express');
+const router = express();
```

Comment:
Remove. Same explanation than the previous file. Also export only router function.